backend/object-management/src/main/kotlin/com/ritense/valtimoplugins/objectmanagement/plugin/ObjectManagementPlugin.kt

```diff
@@ -87,8 +92,22 @@ open class ObjectManagementPlugin(
         description = "Delete an existing Object",
         activityTypes = [ActivityTypeWithEventName.SERVICE_TASK_START]
     )
-    open fun deleteObject() {
+    open fun deleteObject(
+        execution: DelegateExecution,
+        @PluginActionProperty objectUrlVariableName: String,
+        @PluginActionProperty objectManagementConfigurationId: UUID
+    ) {
+        val resolvedValue = valueResolverService.resolveValues(
```

Comment:
Resolving the value is no longer needed. Nowadays, Valtimo resolves the value ootb. So you can just use the `objectUrlVariableName` everywhere